x-pack/test/security_solution_cypress/cypress/e2e/automatic_import/feature_essentials.cy.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ASSISTANT_BUTTON, CREATE_INTEGRATION_LANDING_PAGE } from '../../screens/automatic_import';
+import { login } from '../../tasks/login';
+
+describe(
```

Comment:
The new `automatic_import` folder should have codeowners and those should be the team responsible for the test.

Reply:
Fixed in https://github.com/elastic/kibana/pull/202872/commits/96374b35d5f7773ee4e90c9217c893a686484a30